lib/routes/devices-sessions.js

```diff
@@ -339,41 +339,41 @@ module.exports = (log, db, config, customs, push, devices) => {
         db.sessions(uid)
           .then(sessions => {
             reply(sessions.map(session => {
-              session.id = session.tokenId
-              // if session has a device record
-              session.isDevice = !! session.deviceId
+              const deviceId = session.deviceId
+              const isDevice = !! deviceId
 
-              if (! session.deviceName) {
-                session.deviceName = devices.synthesizeName(session)
+              let deviceName = session.deviceName
+              if (! deviceName) {
+                deviceName = devices.synthesizeName(session)
               }
 
-              session.userAgent = (session.uaBrowser + ' ' + session.uaBrowserVersion).trim()
-
-              if (! session.deviceType) {
-                session.deviceType = session.uaDeviceType || 'desktop'
+              let userAgent
```

Comment:
Is there any place where we log out the full and calculated user agent? Would be interesting to make a dashboard of this and back test that no funky values are being set.